tests/test_relations_hyperlink.py

```diff
@@ -68,6 +69,22 @@ class Meta:
         fields = ('url', 'name', 'nullable_source')
 
 
+# Nullable OneToOne2
+class NullableOneToOneTargetSerializer2(serializers.HyperlinkedModelSerializer):
+    nullable_source2 = serializers.HyperlinkedRelatedField(
+        read_only=True,
+        view_name='nullableonetoonesource2-detail',
+        # This source causes a AttributeError, but seems like the proper value
+        # source='nullable_source.nullable_source2',
+        # This source produces the pk of the nullable_source instance, not nullable_source2
+        source='nullable_source',
```

Comment:
Hi @kevin-brown 
Since Tom closed the issue I'll leave this alone from here. I see now I had committed with the wrong version of `source=` uncommented as I now believe the AttributeError (with `source='nullable_source.nullable_source2' ` to be the real issue. Including the alternative only caused confusion. Sorry for that, just trying to help DRF any way I can as it's so valuable in my current project.

Best regards,
Martin